### PR TITLE
Youtube Clone : Complete Project

### PR DESCRIPTION
**Pull request by ikraamg**
## Youtube Clone

Live Preview LInk: https://rawcdn.githack.com/imahnama/Youtube-replica/bf5cae43186ac1daace6ae73ba8f98c315e99522/index.html
In this project we:
-Added Navigation bar
-Embedded a youtube video
-Added sidebar recommended videos(as images)
-Added a comments section
-Code validated with stickler and W3 markup validation service.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+            Beautiful, a gentle beast roaming the infinite skies.
+          </p>
+        </article>
+      </section>
+
+      <aside class="side-content">
+        <p class="side-descriptions">
+          <img
+            width="300"
+            height="200"
```
Comment (euqueme) [inline index.html]: - [ ] put the width and height selectors in the CSS file as there should be no inline styling in the HTML file  (suggestion: put them in your **.side-images class**)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+          />
+          Contributing to Open Source Part II: The Real Way <br />
+          <br />
+          The Odin Project<br />
+          29,985 views
+        </p>
+        <p class="side-descriptions">
+          <img
+            width="300"
+            height="200"
```
Comment (euqueme) [inline index.html]: - [ ] put the width and height selectors in the CSS file as there should be no inline styling in the HTML file  (suggestion: put them in your **.side-images class**)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+          <br />
+          <br />
+          The Odin Project<br />
+          29,985 views
+        </p>
+
+        <p class="side-descriptions">
+          <img
+            width="300"
+            height="200"
```
Comment (euqueme) [inline index.html]: - [ ] put the width and height selectors in the CSS file as there should be no inline styling in the HTML file  (suggestion: put them in your **.side-images class**)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+            alt="thmub3"
+          />Contributing to Open Source Part II: The Real Way <br />
+          <br />
+          The Odin Project<br />
+          29,985 views
+        </p>
+        <p class="side-descriptions">
+          <img
+            width="300"
+            height="200"
```
Comment (euqueme) [inline index.html]: - [ ] put the width and height selectors in the CSS file as there should be no inline styling in the HTML file  (suggestion: put them in your **.side-images class**)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+            alt="thumb4"
+          />Contributing to Open Source Part II: The Real Way <br />
+          <br />
+          The Odin Project<br />
+          29,985 views
+        </p>
+        <p class="side-descriptions">
+          <img
+            width="300"
+            height="200"
```
Comment (euqueme) [inline index.html]: - [ ] put the width and height selectors in the CSS file as there should be no inline styling in the HTML file  (suggestion: put them in your **.side-images class**)

File: index.html [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+          Contributing to Open Source Part II: The Real Way
+          <br />
+          <br />
+          The Odin Project<br />
+          29,985 views
+        </p>
+        <p class="side-descriptions">
+          <img
+            width="300"
+            height="200"
```
Comment (euqueme) [inline index.html]: - [ ] put the width and height selectors in the CSS file as there should be no inline styling in the HTML file  (suggestion: put them in your **.side-images class**)

File: index.html [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
```
Comment (euqueme) [inline index.html]: - [ ] put the width and height selectors in the CSS file as there should be no inline styling in the HTML file  (suggestion: put them in your **.side-images class**)

File: index.html [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
```
Comment (euqueme) [inline index.html]: - [ ] put the width and height selectors in the CSS file as there should be no inline styling in the HTML file  (suggestion: put them in your **.side-images class**)

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @ikraamg and @imahnama  :balloon: :lollipop: 

### Your project looks extremely professional claps to you for your impressive Readme and very descriptive pull request and commits :clap: :clap:  :clap:  also I see you have stickler on already :muscle: 

### There is still some work to do so let's work on it! :point_down: 

## Project Specifics

**Inline Styling**

- [ ] Please remove all inline styling as this should exist only in the CSS file I've commented your code :arrow_double_down: 

**Video**

- [ ] It shows the avatar of the user who uploaded the video

- Yours

image

- Original

image

## Element sizing

**Navbar**

- [ ] There is too much margin or padding on the left, the Youtube logo should be smaller, also put more space between the icons on the right and make the search bar wider

- Yours

image

- Original

image

**Video**

- [ ] The video should be bigger please follow the size when your browser is fully displayed in your computer screen, also it should have less padding and margin on the left and right(it should be closer to the sidebar)

- Yours

image

- Original

image

**Sidebar**

- [ ] The thumbnails on the sidebar should be smaller.

- Yours

image

- Original

image

### That's it! :raised_hands: 

### Please :pray: keep up the good work, make these changes :memo:  and submit back for a code review when you are done :rocket: 

Happy Coding! :woman_technologist: 

Any questions? reach to me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hi team!! :handshake: 

### Thank you for doing all the changes there's only one thing left to do:

**Video is overlapping**

- [ ] The video overlaps with the sidebar please fix this

image

### That's all

Fix this and submit back for a code review

Happy coding! :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Nice work team! @ikraamg :handshake: @imahnama :handshake: 
You did great building this out and implementing changes suggested by @euqueme :+1: Please note, however, that the subscribe button along with all surrounding buttons are now missing. Please rectify this and request another review.

#### [OPTIONAL]
Please consider comparing your navigation styling along with the styling of all other elements, especially buttons to those on the original page. Some improvements can be made :ok_hand: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by davidmrtz-dev** [comment]
### **Pull Request under the review by David Elí [TSE].**

**Comment by davidmrtz-dev** [comment]
### **Status: Approved ✔️ 💯** 
Hi Team, Congratulations for your job!
It looks very good! 
You have successfully implemented changes requested from the previous reviewers!
**Let's merge the branch and move unto the next project. ⏭  😄**

**[TSE:  David Elí]** 👨‍💻 
Github: [@davitomix](https://github.com/davitomix)
Linkedin: [David Elí Martínez García](https://www.linkedin.com/in/davideligarcia/)

**Pull request merged**